res/home.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Riot - Home</title>
+</head>
+<body>
+    <div>Welcome to Riot</div>
+</body>
+</html>
```

Comment:
Tom and Ben are working on it and we'll have something before `new-guest-access` goes live